tests/constraint/test_SE2BoxConstraint.cpp

```diff
@@ -24,7 +24,7 @@ class SE2BoxConstraintTests : public ::testing::Test
   static constexpr size_t NUM_X_TARGETS { 10 };
   static constexpr size_t NUM_Y_TARGETS { 10 };
   static constexpr size_t NUM_SAMPLES { 1000 };
-  static constexpr double DISTANCE_THRESHOLD { 0.15 };
+  static constexpr double DISTANCE_THRESHOLD { 0.8 };
```

Comment:
@jslee02  For the number of samples taken and the resolution, the maximum distance between a sample and a target in this case is ~0.74. increasing the threshold to 0.8 is sensible in my opinion.